src/styles/blocks/page.scss

```diff
@@ -0,0 +1,64 @@
+.page {
+  font-family: "Manrope", sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 22px;
+  color: #1b2129;
+
+  scroll-behavior: smooth;
```

Comment:
The scroll-behavior should be applied to the :root or html element, not the .page class.